maintainers/maintainer-list.nix

```diff
@@ -53,8 +53,7 @@
    More fields may be added in the future, however, in order to comply with GDPR this file should stay as minimal as possible.
 
    When editing this file:
-    * keep the list alphabetically sorted, check with:
-        nix-instantiate --eval maintainers/scripts/check-maintainers-sorted.nix
+    * keep the list alphabetically sorted
```

Comment:
accidental edit? please fix it if so.

Reply:
The script doesn't exist anymore (https://github.com/NixOS/nixpkgs/tree/master/maintainers/scripts), so I figured I'd remove this reference.

Reply:
Separate that out to a different commit, please.